src/python/pants/core/util_rules/external_tool.py

```diff
@@ -89,7 +122,10 @@ def register_options(cls, register):
             type=str,
             default=cls.default_version,
             advanced=True,
-            help=f"Use this version of {cls.name}.",
+            help=f"Use this version of {cls.name}."
+            + (
+                f" Supported versions: {cls.version_constraints}" if cls.version_constraints else ""
```

Comment:
Instead of a single space, preferred to use `\n\n` so that it appears as a new paragraph.